Error Build GitHub Actions

Today the build in GitHub Actions is failing. 

Log error:

```
error: [Errno 2] No such file or directory: '/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/Pygments-2.7.1.dist-info/METADATA'
Error: Process completed with exit code 1.
```
Log complete : https://github.com/kytos/kytos/runs/1288258354?check_suite_focus=true

--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -56,7 +56,7 @@ ptyprocess==0.6.0         # via -r requirements/run.txt, pexpect
 py==1.8.1                 # via pytest, tox
 pycodestyle==2.5.0        # via yala
 pydocstyle==5.1.1         # via -r requirements/dev.in
-pygments==2.6.1           # via -r requirements/run.txt, ipython, sphinx
+pygments==2.7.1           # via -r requirements/run.txt, ipython, sphinx
 pyjwt==1.7.1              # via -r requirements/run.txt
 pylint==2.4.4             # via yala
 pyparsing==2.4.6          # via packaging

--- a/requirements/run.txt
+++ b/requirements/run.txt
@@ -4,7 +4,6 @@
 #
 #    pip-compile --output-file=requirements/run.txt requirements/run.in
 #
-appnope==0.1.0            # via ipython
 backcall==0.1.0           # via ipython
 click==7.1.1              # via flask
 decorator==4.4.2          # via ipython, traitlets
@@ -26,7 +25,7 @@ pexpect==4.8.0            # via ipython
 pickleshare==0.7.5        # via ipython
 prompt-toolkit==3.0.5     # via ipython
 ptyprocess==0.6.0         # via pexpect
-pygments==2.6.1           # via ipython
+pygments==2.7.1           # via ipython
 pyjwt==1.7.1              # via -r requirements/run.in
 python-daemon==2.2.4      # via -r requirements/run.in
 python-engineio==3.12.1   # via python-socketio

